### PR TITLE
Adding link to Doing Business with Us Page

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
Added a link to the diversity statement to the Small Minority-owned Business page.

## Testing

- Visit http://localhost:8000/doing-business-with-us/small-businesses/. Note the new link

## Review

- @tnferrell 
- @anselmbradford 
- @sebworks 
- @jimmynotjim 

## Screenshots

image

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Comment by anselmbradford** [comment]
:+1:

File: cfgov/jinja2/v1/doing-business-with-us/small-businesses/index.html [hunk 1]
```diff
@@ -59,6 +59,14 @@ <h1 data-qa-hook="main-title">Small & Minority Businesses</h1>
                         <span class="icon-link_text">Info for women- and minority-owned businesses</span>
                     </a>
                 </li>
+                <li class="list_item">
+                    <a class="list_link
+                              icon-link
+                              icon-link__download"
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/doing-business-with-us/small-businesses/index.html]: Is this needed since you've already set all `pdf` file extensions?

**Comment by sebworks** [comment]
:+1:

File: cfgov/jinja2/v1/doing-business-with-us/small-businesses/index.html (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/doing-business-with-us/small-businesses/index.html]: Yeah, it applies some sweet mobile styles for mobile lists.

**Pull request merged**